File: MusicShop.Services/Database/Amplifier.cs
using System;
using System.Collections.Generic;

namespace MusicShop.Services.Database
{
    public partial class Amplifier : Product
    {
        public int? Voltage { get; set; }
        public int? PowerRating { get; set; }
        public bool? HeadphoneJack { get; set; }
        public bool? Usbjack { get; set; }
        public int? NumberOfPresets { get; set; }

    }
}

File: UI/musicshop_mobile/lib/models/amplifier/amplifier.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:musicshop_mobile/models/Abstract/product.dart';
import 'package:musicshop_mobile/models/brand/brand.dart';

part 'amplifier.g.dart';

@JsonSerializable()
class Amplifier {
  int? id;
  String? productNumber;
  Brand? brand;
  String? model;
  double? price;
  String? description;
  int? voltage;
  int? powerRating;
  bool? headphoneJack;
  bool? usbjack;
  int? numberOfPresets;
  DateTime? createdAt;
  DateTime? updatedAt;
  String? productImage;

  Amplifier();

  factory Amplifier.fromJson(Map<String, dynamic> json) =>
      _$AmplifierFromJson(json);

  Map<String, dynamic> toJson() => _$AmplifierToJson(this);
}
